Fix caching for Content Versioning (#20153)
directus/directus: docs/self-hosted/config-options.md

| `CACHE_ENABLED`                              | Whether or not data caching is enabled.                                                                                 | `false`                              |
| `CACHE_TTL`<sup>[1]</sup>                    | How long the data cache is persisted.                                                                                   | `5m`                                 |
| `CACHE_CONTROL_S_MAXAGE`                     | Whether to not to add the `s-maxage` expiration flag. Set to a number for a custom value.                               | `0`                                  |

> | `CACHE_AUTO_PURGE`<sup>[2]</sup>             | Automatically purge the data cache on `create`/`update`/`delete` actions, Content Version saves, and the sort utility.  | `false`                              |

Reviewer: I thought about a simpler sentence like this

```suggestion
| `CACHE_AUTO_PURGE`<sup>[2]</sup>             | Automatically purge the data cache on actions that manipulate the data.  | `false`                              |
```

WDYT?

Author: ```suggestion
| `CACHE_AUTO_PURGE`<sup>[2]</sup>             | Automatically purge the data cache on actions that manipulate the data.                                                 | `false`                              |
```

Sounds good 👍 will update it via this suggestion with formatted spacing.

Reviewer: Thankssss 🐍 ❤️
